Editor/AGXUnityEditor/Windows/ConvertPhysXToAGXWindow.cs

```diff
@@ -0,0 +1,917 @@
+
+      m_toggleAllPrefabObjects = new MixedToggle();
+      m_toggleAllPrefabObjects.style.flexGrow = 0;
+      m_toggleAllPrefabObjects.style.flexShrink = 0;
+      header.Add( m_toggleAllPrefabObjects );
+
+      var numAssets = new VisualElement();
+      numAssets.style.flexDirection = FlexDirection.Row;
+      numAssets.style.alignItems = Align.Center;
+      numAssets.style.unityTextAlign = TextAnchor.MiddleLeft;
+      numAssets.style.flexGrow = 1;
+      numAssets.style.justifyContent = Justify.FlexEnd;
+      numAssets.Add( new Label() { text = "Upgradable prefabs: " } );
+
+      var image = new Image() { image = m_statusIcons[ 0 ] };
+      image.style.width = 20;
+      image.style.height = 20;
+      image.style.marginLeft = 5;
+      numAssets.Add( image );
+
+      var label = new Label() { text = $" {(m_prefabPhysXAssets != null ? m_prefabPhysXAssets.Where( a => a.Updatable).Count() : 0)}" };
+      label.style.flexGrow = 0; 
+      label.style.flexShrink = 0;
+      numAssets.Add( label );
+
+      header.Add( numAssets );
+
+      prefabBox.Add( header );
+
+      var scrolledTable = new ScrollView();
+      if (m_prefabPhysXAssets != null)
+        foreach ( var prefabData in m_prefabPhysXAssets ) // Populate rows with prefabs
+          scrolledTable.Add( CreatePrefabRowUI( prefabData ) );
+
+      prefabBox.Add( scrolledTable );
+      return prefabBox;
+    }
+
+    private void Repopulate( bool gatherAssets = true )
+    {
+      GatherSceneObjects();
+      GatherPrefabData();
+      m_tableRows.Clear();
+
+      rootVisualElement.Clear();
+      CreateGUI();
+    }
+
+    private void ConvertObjects( List<AssetData> assets )
+    {
+      foreach ( var asset in assets ) {
+        if ( !asset.Convert || !asset.Updatable )
+          continue;
+
+        if ( asset.physXType == PhysXType.Collider )
+          ConvertCollider( asset );
+
+        if ( asset.physXType == PhysXType.RigidBody ) {
+          ConvertRigidbody( asset );
+          foreach ( var dependentCollider in asset.dependentAssets )
+            ConvertCollider( dependentCollider );
+        }
+
+        EditorUtility.SetDirty( asset );
+      }
+    }
+
+    private void ConvertSelected()
+    {
+      using ( new UndoCollapseBlock( "Convert Selected PhysX Assets" ) ) {
+        ConvertObjects( m_physXAssets );
+      }
+
+      using ( new UndoCollapseBlock( "Convert Selected Prefab Assets" ) ) {
+        foreach ( var prefab in m_prefabPhysXAssets ) {
+          if ( !prefab.Convert )
+            continue;
+
+          GameObject prefabObject = PrefabUtility.LoadPrefabContents(prefab.Path);
+          if ( prefabObject == null )
+            continue;
+
+          var objects = GatherObjects(prefabObject);
+          try {
+            ConvertObjects( objects );
+
+            PrefabUtility.SaveAsPrefabAsset(prefabObject, prefab.Path);
+          }
+          finally {
+            PrefabUtility.UnloadPrefabContents( prefabObject );
+          }
```

Comment:
Prefer using [PrefabUtility.EditPrefabContentsScope](https://docs.unity3d.com/6000.1/Documentation/ScriptReference/PrefabUtility.EditPrefabContentsScope-ctor.html) for these cases.